playbooks/roles/capsule/tasks/devel_install.yml

```diff
@@ -0,0 +1,19 @@
+- name: 'start rails server'
+  command: /home/vagrant/.rvm/bin/rvm ruby-2.2.4 do rails s
```

Comment:
We should probably set this to just ruby if possible to avoid specific version constraints.